### ISSUE TITLE
brew cask install black-screen 404 Not Found

### ISSUE THREAD

**Issue by seenukarthi** [shockone/black-screen#781]
When i try to install black-screen using `brew cask install black-screen` I'm getting the following error.

``` sh
brew cask install black-screen
==> Downloading https://github.com/shockone/black-screen/releases/download/v0.2.48/black-screen-0.2.48-mac.zip

curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'black-screen' with message: Download failed: https://github.com/shockone/black-screen/releases/download/v0.2.48/black-screen-0.2.48-mac.zip
The incomplete download is cached at /Users/karthik/Library/Caches/Homebrew/Cask/black-screen--0.2.48.zip.incomplete
```

**Comment by adidalal** [comment]
Hi there, I'm one of the maintainers of Homebrew Cask - looks like the dev team released version "0.2.48" with tag name "v0.2.53", which is why the download broke.

Fixed in https://github.com/caskroom/homebrew-cask/pull/24862, but in the future, a less confusing naming scheme would be to keep the tag name and version the same (as it used to be)

Thanks @RJHsiao for the heads up!

**Comment by seenukarthi** [comment]
Its working after `brew update`
